### PR TITLE
YLP-2312 Teams and patients in a team should always have a field "monitoringAlertsParameters"

### PR DESCRIPTION
**Pull request by TimCanu**
## Checklist
- [x] PR name contains the Jira ticket number
- [x] Changelog updated
- [x] Code tested (unit tests and integration tests)
- [x] No console errors/warnings in integration tests
- [x] No translation files updated (except for English files)
- [x] Code formatted on every file
- [x] Copyright added on every file
- [x] End-to-end tests successful on CI

### CODE REVIEW

File: packages/yourloops/test/unit/lib/patient/patient.util.test.ts [hunk 1]
```diff
@@ -49,19 +49,19 @@ const defaultPatientFilters = {
...
   ...defaultMonitoringAlerts,
   frequencyOfSevereHypoglycemiaActive: true
 })
-const patientWithNoDataAlert = createPatient('noData', UserInvitationStatus.accepted, undefined, undefined, undefined, undefined, undefined, {
+const patientWithNoDataAlert = createPatient('noData', UserInvitationStatus.accepted, undefined, undefined, undefined, undefined, {
   ...defaultMonitoringAlerts,
   nonDataTransmissionActive: true
 })
-const noAlersPatient = createPatient('nothing', UserInvitationStatus.accepted, undefined, undefined, undefined, undefined, undefined, defaultMonitoringAlerts)
+const noAlersPatient = createPatient('nothing', UserInvitationStatus.accepted, undefined, undefined, undefined, undefined, defaultMonitoringAlerts)
```
Comment (celia-rouis) [inline packages/yourloops/test/unit/lib/patient/patient.util.test.ts:64]:
```suggestion
const noAlertsPatient = createPatient('nothing', UserInvitationStatus.accepted, undefined, undefined, undefined, undefined, defaultMonitoringAlerts)
```

File: locales/en/yourloops.json [hunk 2]
```diff
@@ -339,6 +339,7 @@
   "refresh": "Refresh",
   "remaining-time": "Remaining time",
   "remote-monitoring": "Remote monitoring",
+  "remote-monitoring-not-implemented": "Remote monitoring is not implemented yet",
```
[the remark this answers is not in the document]
Comment (TimCanu) [reply locales/en/yourloops.json:342]: I don't think that this is expensive (the whole translation process isn't that big) so I can't say if it is worth it or not. But as you think that it is not worth it and I don't mind it so much, I will remove it

**Review by celia-rouis — APPROVED** [review]

**Pull request merged**